content/resource/book.md

```diff
@@ -0,0 +1,50 @@
+---
+title: book
+menu: main
+---
+
+Daniel Jackson's book Software Abstractions offers a gentle and detailed introduction to modeling
+and analysis in Alloy through a sequence of short examples.
+<!--more-->
+
+have to fix image sizes here
+![book](/image/cover.jpg)![book](/image/cover-japanese.jpg)
+
+<div style="float:right;width:200px; margin-left:20px;">
+<a href="https://mitpress.mit.edu/books/software-abstractions"><img  src="static/image/cover.jpg"/></a>
+<br/>
+<a href="https://www.amazon.co.jp/%E6%8A%BD%E8%B1%A1%E3%81%AB%E3%82%88%E3%82%8B%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2%E8%A8%AD%E8%A8%88-Alloy%E3%81%A7%E3%81%AF%E3%81%98%E3%82%81%E3%82%8B%E5%BD%A2%E5%BC%8F%E6%89%8B%E6%B3%95-Daniel-Jackson/dp/4274068587/ref=sr_1_4?ie=UTF8&qid=1503508238&sr=8-4&keywords=software+abstractions"><img  style="margin-top:20px" src="image/cover-japanese.jpg"/></a>
```

Comment:
I *believe* Hugo has an image shortcode. Not sure if it's worth using because it adds more mental overhead to contributing, but I might show you what it looks like at some point

Reply:
yes, you'll see it above this. i had trouble getting it to work because of paths. i wrote a few lines about this in the google doc: 
— Not sure how to get images to show. If I put an image in /static/image/foo.jpg, and refer to it as ![foo](/image/foo.jpg), it fails to load locally unless I remove the suffix from the baseURL. Problem is that the server places it at /image/foo.jpg, but the server runs without the baseURL. Surely you don’t need to change the baseURL to run it locally?
— Not sure how to ensure that book cover images retain their size and don’t get expanded. Can set a CSS rule for img, but then that screws up all images.